backend/search-module/controllers/searchController.ts

```
@@ -0,0 +1,40 @@
+
+		console.log("searching content")
+		try {
+			const response = await SearchService.searchContent(searchText, contentStartingPoint);
+			res.status(200).json(response);
+		} catch (error) {
+			console.error(`Error searching: ${error}`);
+			throw Error(error)
+		}
+
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)